transport/matrix/layer.py

```diff
@@ -77,34 +60,44 @@ def __init__(self, config: Dict[str, Any]):
                                  light_client["address"])
                         continue
 
-                matrix_config = config["transport"]["matrix"]
-                matrix_config["current_server_name"] = current_server_name
+                config = config["transport"]["matrix"]
+                config["current_server_name"] = current_server_name
                 auth_params = {
                     "light_client_password": light_client["password"],
                     "light_client_display_name": light_client["display_name"],
                     "light_client_seed_retry": light_client["seed_retry"]
                 }
                 light_client_transport = MatrixLightClientTransportNode(
                     light_client['address'],
-                    matrix_config,
+                    config,
                     auth_params,
                 )
 
-                self._light_clients.append(light_client_transport)
-
-            self._full_node: TransportNode = MatrixTransportNode(config["address"], config["transport"]["matrix"])
-
+                result.append(light_client_transport)
+            return result
         except RaidenError as ex:
             click.secho(f"FATAL: {ex}", fg="red")
             sys.exit(1)
 
-    @property
-    def full_node(self) -> TransportNode:
-        return self._full_node
+    def _prepare_config(self, config):
```

Comment:
although this method is internal, i think a comment on it would be helpful

Reply:
ive just moved the original raiden code to a function in order to rehuse it both from light client and fn transport creation

Reply:
thanks